src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -165,6 +165,7 @@ export default class DefaultContentShareController
         }
         const stream = this.mediaStreamBroker.mediaStream;
         if (stream.getVideoTracks().length > 0) {
+          this.contentAudioVideo.logger.info('Stream has video tracks');
```

Comment:
To distinguish from the regular audio-video logs, include the term "content share." e.g., `The content share stream has ${stream.getVideoTracks().length} video track(s)`

Reply:
Added this log line.